### ISSUE TITLE
string match fails with some letters with diacritics

### ISSUE THREAD

**Issue by Piscium** [quodlibet/quodlibet#1654]
When searching for songs, an 'a' will find a song with 'a' or 'A' in the name, and 'á' will find songs with 'a' in the name. So far so good.

Bug: a search with 'á' is *not* finding songs with 'Á' in the name.

Example: I searched for a Argentinian tango named 'Buscándote'. I had a m4a track with 'BUSCANDOTE' in the name and it did not find it.

**Comment by lazka** [comment]
Only the base characters will match different variants. 'á' only matches 'á' and 'Á' and not 'a' or 'A'.

I later noticed that this is included in the unicode standard and called [Asymmetric Search](http://unicode.org/reports/tr10/#Asymmetric_Search)

(your example doesn't match your bug description, so I'm not sure if I missed something??)

**Comment by Piscium** [comment]
You are right in that my bug description is wrong. I have QL configured to search for file names and I had the letter I was searching for in the folder name somewhere and that confused me.

Anyway I read that Asymmetric Search link you provided. I think in my own use case I would benefit from a search more wide-ranging than Asymmetric Search as described. Would it be possible for a search for a 'á' also find 'a' and 'A' - even if at the cost of some configuration option?

My use case is that I have thousands of tango songs (with Spanish names, of course) and there is complete lack of consistency with respect to diacritics (as well as misspellings, using diacritics when there should be none, for example) as the songs came from different sources, so it would be helpful in my use case if all the letters 'a', 'A', 'á' and 'Á' were treated as being the same.

**Comment by Piscium** [comment]
Lazka, I think it is better to close this issue because it is a false issue.

I could create a new issue asking for an enhancement if you think it is a good idea.

**Comment by lazka** [comment]
You can also change the title; whatever you prefer.

A configuration option should work.

**Comment by Piscium** [comment]
Done as you suggested, thanks. Yes, a configuration option is a good idea because enhanced string matching may not suit everybody.
